e2e/nodes/test_summarizer.py

```diff
@@ -104,16 +107,12 @@ def test_summarization_one_summary(summarizer):
     assert EXPECTED_ONE_SUMMARIES[0] == summarized_docs[0].meta["summary"]
 
 
-@pytest.mark.integration
-@pytest.mark.summarizer
-@pytest.mark.parametrize(
-    "retriever,document_store", [("embedding", "memory"), ("bm25", "elasticsearch")], indirect=True
-)
-def test_summarization_pipeline_one_summary(document_store, retriever, summarizer):
-    document_store.write_documents(SPLIT_DOCS)
+@pytest.mark.parametrize("retriever", ["bm25", "embedding"], indirect=True)
```

Comment:
I'm not sure we need to parametrize it as we're using all values. Or is it necessary because of `indirect=True`?

Reply:
Not sure what you mean, I run the same test twice passing a `retriever` fixture that's initialised differently at each invocation